src/scripts/components/time-slider/time-slider.styl

```diff
@@ -2,42 +2,120 @@
 
 $range-height = 5px
 
+boxShadow()
+  box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.2), 0px 3px 4px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14)
+
+thumb()
+  boxShadow()
+
+  width: 16px
+  height: 16px
+  border-radius: 50%
+  background: $main
+  cursor: pointer
+
+compareThumb()
+  width: 10px
+  height: 25px
+  border-radius: emCalc(4px)
+
```

Comment:
No need for the extra new line when using mixins

Reply:
That happens automatically. Maybe it's a lint config.

Reply:
Could you check if we can deactivate it in the stylint settings?